docs/ui/components/Home/sections/DiscoverMore.tsx

```diff
@@ -45,13 +45,13 @@ export function DiscoverMore() {
           <OfficeHoursImage />
           <RawH3 className="!text-palette-yellow11 !font-bold">Join us for Office Hours</RawH3>
           <P className="!text-palette-yellow11 !text-xs max-w-[28ch]">
-            Get answers to your questions and get advice from the Expo team.
+            Ask the Expo Team questions on our Discord stage.
           </P>
           <HomeButton
             className="bg-palette-yellow11 border-palette-yellow11 text-palette-yellow2 hocus:bg-palette-yellow11"
-            href="https://us02web.zoom.us/meeting/register/tZcvceivqj0oHdGVOjEeKY0dRxCRPb0HzaAK"
+            href="https://chat.expo.dev"
             rightSlot={<ArrowUpRightIcon className="text-palette-yellow2 icon-md" />}>
-            Register
+            Check Discord for Upcoming Events
```

Comment:
Nit, the button label seems to be a bit length, can we shorten it a bit?

An example"

```suggestion
            See Upcoming Events
```

Reply:
The tricky thing is that the link doesn't actually take you to upcoming events

Reply:
You can use the direct permalink to the stage channel instead:
* https://discord.com/channels/695411232856997968/1136672439980544071